test/e2e/e2e_node_util.js

```diff
@@ -75,6 +76,7 @@ export function e2eNodeKeyRefreshAddTest (keyFormat, testName, mode, releaseTag
     }, defaultTimeout)
 
     afterAll(async () => {
+      await getNodeLogs(k8)
```

Comment:
I think it would be useful to also add this into:
- [ ] `network.test.mjs`
- [ ] `account.test.mjs`
- [ ] `cluster.test.mjs`
- [ ] `relay.test.mjs`

Reply:
Updated